pbspro.spec

```diff
@@ -543,7 +535,21 @@ fi
 %exclude %{pbs_prefix}/unsupported/*.pyc
 %exclude %{pbs_prefix}/unsupported/*.pyo
 %exclude %{_unitdir}/pbs.service
+
+%if %{with ptl}
+%files %{pbs_ptl}
+%defattr(-,root,root, -)
+%dir %{ptl_prefix}
+%{ptl_prefix}/*
+%{_sysconfdir}/profile.d/ptl.csh
+%{_sysconfdir}/profile.d/ptl.sh
+%endif
+
 %changelog
+* Thu Dec 13 2018 Michael Karo <mike0042@gmail.com> - 1.26
+- Bring pbspro.spec.in and pbspro.spec back in sync
```

Comment:
I don't think this really belongs in the changelist for the spec file.  This isn't a change to the spec file, it is a change in the repo.  The change that is being made to the spec file is what was the original change.

Reply:
I like us having a changelog, but I think we should only put things in it that are changes to the spec file.  The fact that we have a pbspro.spec.in and pbspro.spec in our repo doesn't really have anything to do with the inner workings of the spec file itself.  I'd rather you take it out.  The real place for it is probably in your git log message if you were to provide more than one line.  We usually don't do that though.